### PR TITLE
Add deliver_to_functionality tests

### PR DESCRIPTION
**Pull request by IrinaPodoliak**

### CODE REVIEW

**Review by Iggelds — APPROVED** [review]
good job!

**Comment by malmicha** [comment]
Approved!

File: src/test/java/DeliverToTests.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        WebElement popupApplyButton = webDriver.findElement(By.xpath("//*[@id=\"GLUXZipUpdate\"]/span/input"));
+        popupApplyButton.click();
+        WebElement popupContinueButton = new WebDriverWait(webDriver, Duration.ofSeconds(10))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.xpath("/html/body/div[5]/div/div/div[2]/span/span")));
+        popupContinueButton.click();
+
+        webDriver.get("https://amazon.com/");
+
+        WebElement deliverToValue = webDriver.findElement(By.id("glow-ingress-line2"));
+        Assert.assertTrue(deliverToValue.getText().contains("Montgomery"));
```
Comment (juliaHrabovska) [inline src/test/java/DeliverToTests.java:64]: It is better to add a string that explains why the test failed. Same for other

File: src/test/java/DeliverToTests.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        deliverToButton.click();
+
+        WebElement popupInput = new WebDriverWait(webDriver, Duration.ofSeconds(10))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.id("GLUXZipUpdateInput")));
+        popupInput.click();
+        popupInput.sendKeys("36104");
+        WebElement popupApplyButton = webDriver.findElement(By.xpath("//*[@id=\"GLUXZipUpdate\"]/span/input"));
+        popupApplyButton.click();
+        WebElement popupContinueButton = new WebDriverWait(webDriver, Duration.ofSeconds(10))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.xpath("/html/body/div[5]/div/div/div[2]/span/span")));
```
Comment (juliaHrabovska) [inline src/test/java/DeliverToTests.java:58]: It is not recommended to use absolute path. Please change it next commit

Comment (juliaHrabovska) [reply src/test/java/DeliverToTests.java:58]: Same for others

File: src/test/java/DeliverToTests.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        WebElement notebooksCategoryButton = webDriver.findElement(By.xpath("/html/body/div[1]/div[2]/div[2]/div[2]/div[5]/div/div[2]/div[1]/div[1]/a"));
+        notebooksCategoryButton.click();
+
+        WebElement selectedElementFromCategory = webDriver.findElement(By.xpath("/html/body/div[1]/div[2]/div[1]/div[1]/div/span[3]/div[2]/div[2]/div/div/div/div/div/div"));
+        selectedElementFromCategory.click();
+        WebElement shippingDetails = webDriver.findElement(By.xpath("//*[@id=\"contextualIngressPtLabel_deliveryShortLine\"]/span[2]"));
+        Assert.assertTrue(shippingDetails.getText().contains("Canada"));
+
+    }
+
```
Comment (juliaHrabovska) [inline src/test/java/DeliverToTests.java:108]: Extra spaces

**Pull request merged**